src/main/scala/scalaz/parsers/alt/AlternativeClass.scala

```diff
@@ -1,13 +1,17 @@
-package scalaz.parsers
+package scalaz.parsers.alt
```

Comment:
File naming convention I used, e.g. `alternative.scala`, is from scalaz-8.
scalaz-7 convention would be `Alternative.scala`.
`AlternativeClass.scala` is unique to us 😉